modules/blocks.php

```diff
@@ -190,3 +190,38 @@ function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow', $dependencies );
 	return $content;
 }
+
+/**
+ * Visited Block.
+ */
+jetpack_register_block( 
+	'visited', 
+	array(
+		'render_callback' => 'jetpack_visited_block_render',
+	)
+);
+
+/**
+ * Visited block dependency declaration.
+ *
+ * @param array  $attributes - Array containing the map block attributes.
+ * @param string $content - String containing the visited block content.
+ *
+ * @return string
+ */
+function jetpack_visited_block_render( $attributes, $content ) {
+	Jetpack_Gutenberg::load_assets_as_required( 'visited' );
+
+	$count = intval( $_COOKIE[ 'wp-visit-tracking' ] );
```

Comment:
I would recommend adding a check before this, to avoid notices like this in the admin for example:

```
[06-Feb-2019 14:36:10 UTC] PHP Notice:  Undefined index: wp-visit-tracking in /public/wp-content/plugins/jetpack-dev/modules/blocks.php on line 215
```

Reply:
Makes sense, thanks for the heads up!